cmd/cmdutils.go

```diff
@@ -103,3 +105,123 @@ func validateStoragePath(client *occlient.Client, storagePath, componentName, ap
 	}
 	return nil
 }
+
+// printStorageInComponent prints all the storage in a given component of the application
+func printStorageInComponent(client *occlient.Client, applicationName string, componentName string) {
+	//these flags are initially false, will be true if we find any mounted or unmounted storage respectively
+	hasMounted := false
+	hasUnmounted := false
+
+	//defining the column structure of the table
+	tabWriterMounted := tabwriter.NewWriter(os.Stdout, 5, 2, 3, ' ', tabwriter.TabIndent)
+	tabWriterUnmounted := tabwriter.NewWriter(os.Stdout, 5, 2, 3, ' ', tabwriter.TabIndent)
+
+	//create headers of mounted and unmounted storage table
+	fmt.Fprintln(tabWriterMounted, "NAME", "\t", "SIZE", "\t", "PATH")
+	fmt.Fprintln(tabWriterUnmounted, "NAME", "\t", "SIZE")
+
+	storageListMounted, err := storage.ListMounted(client, componentName, applicationName)
+	checkError(err, "Could not get mounted storage list")
+
+	//iterating over all mounted storage and put in the mount storage table
+	if len(storageListMounted) > 0 {
+		hasMounted = true
+		for _, mstorage := range storageListMounted {
+			fmt.Fprintln(tabWriterMounted, mstorage.Name, "\t", mstorage.Size, "\t", mstorage.Path)
+		}
+	}
+
+	storageListUnmounted, err := storage.ListUnmounted(client, applicationName)
+	checkError(err, "Could not get unmounted storage list")
+
+	//iterating over all unmounted storage and put in the unmount storage table
+	if len(storageListUnmounted) > 0 {
+		hasUnmounted = true
+		for _, ustorage := range storageListUnmounted {
+			fmt.Fprintln(tabWriterUnmounted, ustorage.Name, "\t", ustorage.Size)
+		}
+	}
+
+	//print all mounted storage of the given component
+	if hasMounted {
+		fmt.Printf("The component '%v' has the following storage attached:\n", componentName)
+		tabWriterMounted.Flush()
+	} else {
+		fmt.Printf("The component '%v' has no storage attached\n", componentName)
+	}
+
+	//print all unmounted storage of the application
+	if hasUnmounted {
+		fmt.Printf("\nThe following unmounted storage can be mounted to '%v':\n", componentName)
+		tabWriterUnmounted.Flush()
+	} else {
+		fmt.Printf("\nNo unmounted storage exists to mount to '%v' \n", componentName)
+	}
+}
+
+// printStorageInAllComponent prints all the storage in all the components of the application and project
+func printStorageInAllComponent(client *occlient.Client, applicationName string, projectName string) {
+	componentList, err := component.List(client, applicationName, projectName)
+	checkError(err, "Could not get component list")
+
+	//iterating over all the components in the given aplication and project
+	for _, component := range componentList {
+		componentName := component.Name
```

Comment:
Can we reuse printStorageInComponent() here? Maybe by adding a flag in printStorageInComponent() called printUnmounted to print the unmounted storage part for the component